distributionviewer/core/static/css/chart-config.styl

```diff
@@ -0,0 +1,22 @@
+@import 'lib';
+
+#chart-config {
```

Comment:
IDs are a bit too specific which makes overriding them tricky later. The rest seems good though.